doc/source/api/sequenceannotation.rst

```diff
@@ -0,0 +1,33 @@
+.. _sequence_annotation
```

Comment:
I believe this file is also included as part of the sequence annotation PR #591 - We'll resolve any differences upon merge.